bundles/org.openhab.binding.boschshc/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,281 @@
+		<item-type>Number:Temperature</item-type>
+		<label>Temperature</label>
+		<description>Current measured temperature.</description>
+		<state min="0" max="40" step="0.5" pattern="%.1f °C" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="temperature-rating">
+		<item-type>String</item-type>
+		<label>Temperature Rating</label>
+		<description>Rating of the currently measured temperature.</description>
+	</channel-type>
+
+	<channel-type id="humidity">
+		<item-type>Number</item-type>
+		<label>Humidity</label>
+		<description>Current measured humidity.</description>
+	</channel-type>
+
+	<channel-type id="humidity-rating">
+		<item-type>String</item-type>
+		<label>Rating of current humidity</label>
+		<description>Current measured humidity.</description>
+	</channel-type>
+
+	<channel-type id="energy-consumption">
+		<item-type>Number</item-type>
+		<label>Energy consumption (Wh)</label>
+		<description>Energy consumption of the device.</description>
+	</channel-type>
+
+	<channel-type id="power-consumption">
+		<item-type>Number</item-type>
+		<label>Power consumption (W)</label>
+		<description>Current power consumption of the device.</description>
+	</channel-type>
+
+	<channel-type id="purity">
+		<item-type>Number</item-type>
+		<label>Purity</label>
+		<description>Purity of the air.</description>
+	</channel-type>
+
+	<channel-type id="air-description">
+		<item-type>String</item-type>
+		<label>Description</label>
+		<description>Overall description of the air quality.</description>
+	</channel-type>
+
+	<channel-type id="purity-rating">
+		<item-type>String</item-type>
+		<label>Purity Rating</label>
+		<description>Rating of the air purity.</description>
+	</channel-type>
+
+	<channel-type id="combined-rating">
+		<item-type>String</item-type>
+		<label>Combined Rating</label>
+		<description>Combined rating of the air quality.</description>
+	</channel-type>
+
+	<channel-type id="contact">
+		<item-type>Contact</item-type>
+		<label>Window/Door contact</label>
+		<description>A Bosch window and door contact.</description>
+	</channel-type>
+
+	<channel-type id="latest-motion">
+		<item-type>DateTime</item-type>
+		<label>Latest motion</label>
+		<description>Timestamp of the latest motion.</description>
+	</channel-type>
+
+	<channel-type id="level">
+		<item-type>Rollershutter</item-type>
+		<label>Level</label>
+		<description>Current open ratio (0 to 100).</description>
+		<state min="0" max="100" step="0.5" readOnly="false"/>
+	</channel-type>
+
+	<channel-type id="valve-tappet-position">
+		<item-type>Number</item-type>
+		<label>Valve Tappet Position</label>
+		<description>Current open ratio (0 to 100).</description>
+		<state min="0" max="100" step="1" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="setpoint-temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Setpoint Temperature</label>
+		<description>Desired temperature (in degree celsius).</description>
```

Comment:
The user can also specify °F. The text in brackets should be removed.

Reply:
Changed the description